test/verify/check-system-info

```diff
@@ -710,6 +713,26 @@ machine         : 8561
         b.wait_text('#memory-listing tr:nth-of-type(2) td[data-label=Rank]', "Single rank")
         b.wait_in_text('#memory-listing tr:nth-of-type(2) td[data-label=Speed]', "2400 MT/s")
 
+        # Pretend UEFI and Secure Boot is enabled
+        m.execute("echo -en '\\x06\\x00\\x00\\x00\\x01' > /tmp/SecureBoot-8be4df61-93ca-11d2-aa0d-00e098032b8c")
+        self.write_file("/etc/cockpit/systemd.override.json",
```

Comment:
Okay.  Now I understand.  You're doing this to give the tests a hook to mock in a new value.

How about a bind mount instead?

Reply:
I tried, couldn't create the missing folder structure, if you have any proposal I am up for it. I am sure creating a VM and booting it with EFI enabled is overkill